index.js

```diff
@@ -95,6 +95,18 @@ module.exports = {
                   /components\/scroll-anchor/
                 ];
                 break;
+              case 'non-bg-vi':
+                patterns = [
+                  /services\/scroll/,
+                  /services\/viewport/,
+                  /helpers/,
+                  '**/global-rules.less',
```

Comment:
you need to keep `'**/global-rules.less',`, so remove from this "exclude" list too